### ISSUE TITLE
Table must have Consistent positioning

### ISSUE THREAD

**Issue by lady-yanagi** [kevincobain2000/laravel-erd#27]
`Before`
![beforeBrowserRefresh](https://github.com/kevincobain2000/laravel-erd/assets/72491828/967f0d8b-0102-4af0-9cd8-7eae52926333)
`this is my arrangement.`

I've found that everytime I refresh the browser, the positioning of the table was turn to shuffle. (see the image below)

`After`
![afterBrowserRefresh](https://github.com/kevincobain2000/laravel-erd/assets/72491828/90238147-263d-452d-878b-4f3ea41020e3)

it will better if they put "Save button" so that every time they have changes it will be convenient.

**Comment by kevincobain2000** [comment]
Save button or auto save sounds like an excellent idea

**Comment by Hillcow** [comment]
I was going to create an issue asking for auto save. Otherwise it's all over the place.

**Comment by kevincobain2000** [comment]
Thanks. IDK man when I would push this feature request. I am packed with issues all over on my other repos too.

**Comment by jacobfogg** [comment]
I agree with the save of positioning. That would make this a slam dunk! I am finding that having an ERD helps myself and developer properly visualize the database. What tooling are you using for the layout tools? I would be happy to put some effort myself into the layout tooling as it would streamline a lot of what I am trying to accomplish.
